resotoclient/ca.py

```diff
@@ -1,24 +1,19 @@
 from cryptography.x509.base import Certificate
 from cryptography import x509
 import warnings
-import requests
+import aiohttp
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives import hashes
 from cryptography.hazmat.primitives import serialization
-from cryptography.x509.oid import NameOID
 from resotoclient.jwt_utils import decode_jwt_from_headers
 from jwt.exceptions import InvalidSignatureError
 import logging
 from logging import Logger
-import certifi
```

Comment:
Without certifi does the client still work for people who don't use the Resoto CA and just install their own AWS or Verisign provided certificate?